packages/ui/src/components/BenefitCard/BenefitCard.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Box } from '@chakra-ui/react';
+import { Meta, Story } from '@storybook/react';
+
+import { SolidGitHubIcon } from '../../icons';
+import { BenefitCard, BenefitCardProps } from './BenefitCard';
+
+const meta: Meta = {
+  title: 'Components / BenefitCard',
+  component: BenefitCard,
```

Comment:
💭 You can add the following code here:
```
argTypes: {
    icon: {
        control: false,
    }
}
```
It will disable the possibility to edit this prop from the controls panel. As it's a react component even now we cannot change its value to something else that would work.